addons/discuss-button/addon.json

```diff
@@ -1,12 +1,10 @@
 {
-  "name": "Discuss button",
```

Comment:
The problem with just replacing the old aaddon

Reply:
Yep! That's not a problem! They'll be able to edit it even more than before!

Reply:
Yeah, well is there a way to automatically make the last button "Discuss" if they had the old addon enabled? otherwise they might confused about why the addon isn't workinganymore

Reply:
When they realize it isn't working anymore, they'll look back at the settings page and be thrilled with all the new updates to it.

Reply:
I guess... and they'll be the update notification...

Reply:
I think if this addon is marked as new, the user will be able to easily spot it.

Reply:
that's not a great way of doing it.

Reply:
Yeah I agree. I just don't think it would be cool to keep 2 addons, with one that has way more functionality than the second. If you have a better idea, we can see if W_L will approve